src/Analyzers/VisualBasic/Analyzers/UseIsNotExpression/VisualBasicUseIsNotDiagnosticAnalyzer.vb

```diff
@@ -43,7 +43,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseIsNotExpression
             Dim node = syntaxContext.Node
             Dim syntaxTree = node.SyntaxTree
 
-            ' "x is not Type y" is only available in C# 9.0 and above. Don't offer this refactoring
+            ' "x IsNot ...." is only available in VB 14.0 and above. Don't offer this refactoring
```

Comment:
@CyrusNajmabadi 

https://github.com/dotnet/roslyn/blob/bf26b0607760eabf50ee20e60513b7492decea96/docs/wiki/New-Language-Features-in-VB-14.md

This document states "TypeOf <expression> IsNot <type> operator" as a  new VB14 feature.

I'm not sure, but I feel like the new feature for VB14 is the usage of both TypeOf and IsNot together. So, something like "x IsNot Nothing" might be available before version 14?

Reply:
> I'm not sure, but I feel like the new feature for VB14 is the usage of both TypeOf and IsNot together. So, something like "x IsNot Nothing" might be available before version 14?

possibly.  i don't know if we care.   VB 14 was 5 years ago.